dataset/put-metadata.go

```diff
@@ -97,3 +98,77 @@ func putMetadata(w http.ResponseWriter, req *http.Request, dc DatasetClient, zc
 
 	log.Info(ctx, "put metadata: request successful", log.Data(logInfo))
 }
+
+// PutEditableMetadata updates a given list of metadata fields, agreed as being editable for both a dataset and a version object
+// This new endpoint makes an unique call to the dataset api updating only the relevant metadata fields in a transactional way
```

Comment:
A very small typo here:
```suggestion
// This new endpoint makes a unique call to the dataset api updating only the relevant metadata fields in a transactional way
```